src/triggers/newCandyMachineNfts.ts

```diff
@@ -0,0 +1,51 @@
+
+
+import { JsonMetadata, Metadata } from '@metaplex-foundation/js';
+import { web3 } from '@project-serum/anchor';
+import { ConfirmedSignatureInfo, PublicKey } from '@solana/web3.js';
+import _ from 'lodash';
+
+import { Table } from '../db/db';
+import { MetaFactory } from '../types/metaFactory';
+import { NftFactory } from '../types/nft';
+import { Trigger } from '../types/trigger';
+import { rollPromises } from '../utils/rollingPromises';
+
+export const newCandyMachineNfts: (metaFactory: MetaFactory) => Trigger<undefined> =
+(metaFactory) => async (clients) => {
+  const existingNFTFactories = new Set(
+    (await clients.db.getRecords<NftFactory>(Table.nftFactories, [
+      [
+        'where', ['platformId', metaFactory.platformId]
+      ]
+    ])).map(nftFactory => nftFactory.id));
+
+
+  const allMintAccounts = await clients.solana.getMintAddressesForCandyMachine(new PublicKey(metaFactory.id))
+  const newMintAccounts = allMintAccounts.filter(mintAccount => !existingNFTFactories.has(mintAccount));
+  console.log('new candy machine nfts', newMintAccounts)
+  const chunks = _.chunk(newMintAccounts, 100)
```

Comment:
Should we extract `100` to a well named ENV like `MAX_CANDY_MACHINE_CHUNK_SIZE`? 

Or alternatively use `MAX_INSERT_CHUNK_SIZE`?